src/sentry/static/sentry/app/views/alerts/rules/details/body.tsx

```diff
@@ -455,6 +455,7 @@ class DetailsBody extends React.Component<Props> {
                       </code>
                     ),
                   })}
+                  {(rule?.query || rule?.dataset) && <br />}
                   {(rule?.query || rule?.dataset) &&
```

Comment:
Did you mean to remove this line?

Reply:
the diff you're commenting on intends to add a line. i'm not sure if this is the most idiomatic react syntax but it appears to achieve the desired result.

Reply:
oh duh

Reply:
i previously tried jamming the `<br/>` in with various syntaxes (e.g. with and without parentheses) and no combination appeared to achieve the right results. maybe i didn't try the right syntax though, am open to suggestions to reduce this duplication.